scripts/render/processSvg.mjs

```diff
@@ -1,7 +1,7 @@
 import { optimize } from 'svgo';
 import * as prettier from 'prettier';
 import { parseSync, stringify } from 'svgson';
-import DEFAULT_ATTRS from '../../tools/build-icons/render/default-attrs.json' assert { type: 'json' };
+import DEFAULT_ATTRS from '../../tools/build-icons/render/default-attrs.json' with { type: 'json' };
```

Comment:
I wonder how this worked for anyone previously, I had to make this change for the script to run.

Reply:
Ah, I just found out that this was valid syntax in v20 and earlier